tests/swagger.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test, expect } from '@playwright/test';
+
+test('swagger ui renderer', async ({ page }) => {
+  await page.goto('/');
+
+  await expect(page).toHaveTitle(/API Browser/);
+  await expect(page.getByText('Authorize')).toBeVisible();
+});
```

Comment:
add a simple minimal test for swagger-ui-ivy ... in order to get fast feedback on braking changes supplied by dependabot updates